rosidl_typesupport_opensplice_cpp/resource/msg__type_support.cpp.template

```diff
@@ -380,7 +380,9 @@ static message_type_support_callbacks_t callbacks = {
   "@(spec.base_type.type)",
   &register_type__@(spec.base_type.type),
   &publish__@(spec.base_type.type),
-  &take__@(spec.base_type.type)
+  &take__@(spec.base_type.type),
+  nullptr,  // convert ros to dds (handled differently for C++)
```

Comment:
I'm exactly not sure about this, but could we scratch these fields from the `message_type_support_callbacks_t` struct for rosidl_typesupport_opensplice_cpp, since they aren't filled in for C++, and the C package uses its own definition of this struct?

It's defined in include/rosidl_typesupport_connext_cpp/message_type_support.h:

https://github.com/ros2/rmw_opensplice/blob/master/rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/message_type_support.h